site/logging.md

```diff
@@ -200,6 +200,7 @@ To connect using TLS, you should configure ssl options:
 <pre class="sourcecode ini">
 log.syslog = true
 log.syslog.transport = tls
+log.syslog.protocol = rfc5424
```

Comment:
You can omit protocol configuration If choosing `tls`. It will be `rfc5424 ` by default. But I agree with more explicit docs.